docs/automated-process.md

```diff
@@ -0,0 +1,30 @@
+# Automated Ansible Release Process
+
+## Preamble
+
+This document describes the (mostly) automated ansible community package release process.
```

Comment:
I'd add a link here to the docs for the manual process.

Reply:
Inserted in 5ad0ad71749cb2907b8cdd6115246bbd5d9b8045.